packages/ariakit/src/checkbox/__examples__/checkbox-group/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Checkbox, useCheckboxState } from "ariakit/checkbox";
+import { Group, GroupLabel } from "ariakit/group";
+
+import "./style.css";
```

Comment:
Nitpick just to keep it consistent with other examples. We may enforce a style with eslint in the future (I just don't know how to do it 😅).

```suggestion
import { Group, GroupLabel } from "ariakit/group";
import "./style.css";
```

Reply:
Did the changes here.

I use https://github.com/lydell/eslint-plugin-simple-import-sort to group my imports in my project as per my style.
Checkout this package, if it seems suitable I can raise a PR.

Example Eslint config in my app,